lib/symmetric-key.js

```diff
@@ -39,7 +40,14 @@ exports.generateKey = async ({keyId, type}) => {
   };
   try {
     await database.collections.ssm.insertOne(record, database.writeOptions);
-    return {'@context': _constants.SECURITY_CONTEXT_V2_URL, id: keyId, type};
+    return {
+      '@context': [
+        WEBKMS_CONTEXT_URL,
+        _constants.KEY_TYPES.symmetric.get(type)
+      ],
+      id: keyId,
+      type
+    };
```

Comment:
We're returning `@context` for symmetric keys here. Do we do the same for asymmetric ones with the API changes here? Please confirm we're consistent. Thanks!

Reply:
@dlongley ahh good point. We are not returning contexts on key generation for asymmetric keys. Will fix..

Reply:
@dmitrizagidulin did this get addressed?

Reply:
@mattcollier it did get addressed, yes, thank you for double-checking!

Reply:
@mattcollier it did, thank  you!